tsconfig.base.json

```diff
@@ -528,8 +528,6 @@
       "@kbn/core-usage-data-server-internal/*": ["packages/core/usage-data/core-usage-data-server-internal/*"],
       "@kbn/core-usage-data-server-mocks": ["packages/core/usage-data/core-usage-data-server-mocks"],
       "@kbn/core-usage-data-server-mocks/*": ["packages/core/usage-data/core-usage-data-server-mocks/*"],
```

Comment:
What were these 2 lines even doing?  I've no idea.

Reply:
I mean the deleted lines

Reply:
It is auto-generated file. It was update by bootstrap command. I'm not aware of the purpose either.